src/index.js

```diff
@@ -22,7 +22,8 @@ class WebpackPwaManifest {
       fingerprints: true,
       ios: false,
       publicPath: null,
-      includeDirectory: true
+      includeDirectory: true,
+      crossorigin: null
```

Comment:
do you want to update [`index.d.ts`](https://github.com/arthurbergmz/webpack-pwa-manifest/blob/master/index.d.ts) as well?

Reply:
do you want to also update the object of presets in [`README.md`](https://github.com/arthurbergmz/webpack-pwa-manifest/blob/master/README.md#api)?

Reply:
can you update the defaults in `README.md`?